src/databricks/labs/ucx/contexts/application.py

```diff
@@ -166,7 +166,9 @@ def scim_entitlements_support(self):
     @cached_property
     def secret_scope_acl_support(self):
         # Secret ACLs are not used much in tests, so skipping include_object_permissions
-        return SecretScopesSupport(self.workspace_client)
+        return SecretScopesSupport(
+            self.workspace_client, include_object_permissions=self.config.include_object_permissions
```

Comment:
Should only the object permissions that start with `secrets:` be passed on? Or all?

Reply:
`StaticListing` handles the filtering, clarified in comments